TIL/topics/dart_basics/ModelClass.md

```
@@ -0,0 +1,58 @@
+## ModelClass
+---
+### 데이터 소스를 앱에서 필요한 형태로 변환한 데이터를 담는 클래스이며 별도의 기능을 가지지 않고 속성에 대한 데이터를 조회할 수 있다.
+- 비슷하게 불리는 용어: 도메인 모델, Entity, DTO, POJO, VO, 데이터 클래스
+- 모델링 방법
+	- DDD(Domain Driven Design)
+	- ORM(Object-relational mapping)
+- 불변 객체
+	- Dart의 경우 생성자로 받는 인자의 필드 값이 모두 final 이면 const  생성자 생성 가능
+		```dart
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)